test/resources/mnist/mnist_estimator.py

```diff
@@ -22,7 +24,7 @@ def cnn_model_fn(features, labels, mode):
   # Padding is added to preserve width and height.
   # Input Tensor Shape: [batch_size, 28, 28, 1]
   # Output Tensor Shape: [batch_size, 28, 28, 32]
-  conv1 = tf.layers.conv2d(
+  conv1 = tf.compat.v1.layers.conv2d(
```

Comment:
same comment as sdk, should we rather change it to a tf-2.0 style API rather than using compat.v1 ?